web/src/test/java/no/nav/modiapersonoversikt/service/sfhenvendelse/SfHenvendelseServiceImplTest.kt

```diff
@@ -60,11 +60,11 @@ internal class SfHenvendelseServiceImplTest {
     @Test
     internal fun `skal fjerne kontorsperrede henvendelser`() {
         every { ansattService.hentAnsattFagomrader(any(), any()) } returns setOf("DAG", "OPP")
-        every { arbeidsfordeling.hentGTnummerForEnhet(any()) } returns listOf(
-            EnhetsGeografiskeTilknytning().also {
-                it.enhetId = 5678
-                it.geografiskOmraade = "005678"
-            }
+        every { arbeidsfordeling.hentGeografiskTilknyttning(any()) } returns listOf(
+            EnhetGeografiskTilknyttning(
+                enhetId = "5678",
+                geografiskOmraade = "005678"
```

Comment:
Kjem GT som kun 6 siffera kode i rest-tjenestene mot variasjonen 4 for kommune og 6 for bydel?

Reply:
Geografiskområde kan være 4-siffer, e.g 3805 Larvik som behandles av enheten 5303 NAV Larvik.

Reply:
Sjekket api-responsen for kallet som skjer mot norg ved kall til `arbeidsfordeling.hentGeografiskTilknyttning(5303)` og fikk følgende tilbake:
```
{
  "geografiskOmraade": "3805"
  // Litt andre felter med interne enhetId'er osv
}
```

Tilsvarende om jeg gjør kallet for 0316 NAV Gamle Oslo returneres 030101 som geografiskområde.